app/controllers/demo_mode/sessions_controller.rb

```diff
@@ -58,6 +59,7 @@ def render_signinable_json
           status: @session.status,
           username: @session.signinable_username,
           password: @session.signinable_password,
+          metadata: @session.signinable_metadata,
```

Comment:
I am reading this PR correctly that if I wanted to get back the `persona_name` or just about any data on the use I could do that through this? Just trying to understand how this API works.

Reply:
Yeah, we can pretty much expose anything available within the signinable